frontend/packages/operator-lifecycle-manager/src/style.scss

```diff
@@ -296,3 +296,47 @@ $co-affinity-row-margin: 15px;
 .olm-descriptor__title {
   display: inline-block;
 }
+
+//TODO: This will change the bg color everywhere, need to make it only on the install screen
+#content-scrollable {
+  background-color: #f0f0f0;
+}
+
+.co-m-list .pf-l-bullseye {
+  margin: 0px 0 0 0;
+}
+
+.pf-c-card__body.co-clusterserviceversion-install__gray_background {
+  background-color: #f0f0f0;
+}
+
+#operator-install-page .pf-c-card {
+  margin-bottom: 5px;
+  // max-width: 75%;
+  margin-top: auto;
+}
+
+#operator-install-page .pf-c-card p {
+  max-width: 50%;
```

Comment:
This does reduce the width of the paragraph, but does nothing for the card width